app/src/main-process/main.ts

```diff
@@ -285,10 +285,6 @@ function onDidLoad(fn: OnDidLoadFn) {
   if (onDidLoadFns) {
     onDidLoadFns.push(fn)
   } else {
-    if (!mainWindow) {
-      createWindow()
-    }
-
     fn(mainWindow!)
```

Comment:
Can we skip running the function if we don't have a mainwindow? I know this is nitpicking but with #1562 we'll have two places that null out a window.